Resources/Prototypes/Entities/Clothing/Uniforms/ship_vs_ship.yml

```diff
@@ -93,4 +93,6 @@
   - type: Sprite
     sprite: Clothing/Uniforms/Jumpsuit/ce_syndie.rsi
   - type: Clothing
-    sprite: Clothing/Uniforms/Jumpsuit/ce_syndie.rsi
\ No newline at end of file
+    sprite: Clothing/Uniforms/Jumpsuit/ce_syndie.rsi
+  - type: Reflect
+    reflectProb: 0.5
```

Comment:
Odd choice for making a syndie jumpsuit reflect. What's the reason for it?

Reply:
item desc says it has a reflective vest

Reply:
I think that means an emergency reflective vest a roadworker wears, not the armory one.

Reply:
my sleep-deprived ass completely missing this